.changeset/sweet-rivers-dress.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/prompting-ui': patch
+---
```

Comment:
not sure this changeset is needed when `examples/prompting-ui`is updated.

Reply:
I am also was not sure, but saw that other example packages also has changeset file like
https://github.com/SAP/open-ux-tools/blob/main/examples/odata-cli/CHANGELOG.md
-> https://github.com/SAP/open-ux-tools/pull/2157/files#diff-4d640317d13b1ce460db0d741e3049ada8421ba4f0657fdd1f1d28eb6e0b836aR3